frontend/src/components/mainPage/scene/subcomponents/SceneObjectsPanel.vue

```diff
@@ -44,3 +47,10 @@
         },
     });
 </script>
+
```

Comment:
W projekcie mamy skonfigurowanego prettiera (narzędzie do automatycznego formatowania). Przy robieniu PR automatycznie sprawdzane jest czy zgadza się formatowanie czy nie, w tym przypadku wyskoczył błąd formatowania. Aby to odpalić wykonaj polecenie `npm run format` w folderze frontend.